security-whitepaper/cloud-native-security-whitepaper.md

```diff
@@ -307,15 +307,13 @@ Hardening of infrastructure and workloads should be supported by comprehensive t
 
 ##### Static Analysis and Security Testing
 
-Static analysis of IaC, application manifests, and software code may cover linting, identifying misconfigurations and vulnerability scanning. IaC code should be subject to the same pipeline policy controls as are application workloads.
+Static analysis of IaC, application manifests, and software code provides linting and identification of misconfigurations and vulnerabilities. IaC code should be subject to the same pipeline policy controls as are application workloads.
```

Comment:
It’s important to note that static analysis of software code will examine only that specific component (microservice for example) without any context of the surrounding components. Therefore, the ability of such a tool to provide accurate identification of vulnerabilities across the application is very limited. Unfortunately, such a static analysis will result in a high rate FN and FP in a cloud-native environment (if we want to be more specific - in a microservices architecture)

Reply:
@RonVider  i think the distinction is important enough to call it out.  made update
```suggestion
Static analysis of IaC, application manifests, and software code provide linting, identification of misconfigurations, and vulnerabilities of the specific component and will not include context of the surrounding components in analysis.  It is important to note that individual analysis is important, but should not be the only form of analysis. IaC code should be subject to the same pipeline policy controls as are application workloads.
```